gdcdatamodel/models/versioning.py

```diff
@@ -1,6 +1,7 @@
 import os
 import uuid
 
+import six
```

Comment:
we should add `six` to setup.py.  it's currently available because it's a transitive dependency.